### PR TITLE
CDPS-736: ModSecurity updates

### PR DESCRIPTION
**Pull request by davidmiddletonmoj**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -47,6 +47,13 @@ generic-service:
       SecRuleUpdateTargetById 942150 "!ARGS:/outcomeComment/"
       SecRuleUpdateTargetById 942230 "!ARGS:/.*description/"
       SecRuleUpdateTargetById 942410 "!ARGS:/outcomeComment/"
+      # Disable rule 932110 as it only applies to Windows host servers
```
Comment (davidmiddletonmoj) [inline helm_deploy/values-dev.yaml:50]: There is conflicting advice online about where to place some of these rules - either at the end as here, or directly after the `SecRuleEngine On` - Would welcome any input on this.

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -47,6 +47,13 @@ generic-service:
...
+      SecRuleRemoveById 932110
+      # Disable rule 942360 (SQL Injection) for url /api/bookings/:bookingId/alert/:alertId
+      SecRule REQUEST_URI "@rx ^\/api\/bookings\/[\w|\d]*\/alert\/[\w|\d|-]*" "id:800001,phase:1,pass,t:none,nolog,ctl:ruleRemoveById=942360"
```
Comment (davidmiddletonmoj) [inline helm_deploy/values-dev.yaml:53]: Have used `800001` and `800002` as IDs as I think these should be unique

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (petergphillips) [reply helm_deploy/values-dev.yaml:50]: don't really mind - whatever works.  Best approach is to normally manually edit the ingress in dev to see if the changed rules are working before then raising PR - happy to help with this.

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -47,6 +47,13 @@ generic-service:
       SecRuleUpdateTargetById 942150 "!ARGS:/outcomeComment/"
       SecRuleUpdateTargetById 942230 "!ARGS:/.*description/"
       SecRuleUpdateTargetById 942410 "!ARGS:/outcomeComment/"
+      # Disable rule 932110 as it only applies to Windows host servers
+      SecRuleRemoveById 932110
```
Comment (petergphillips) [inline helm_deploy/values-dev.yaml:51]: was this rule being triggered then?

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (petergphillips) [reply helm_deploy/values-dev.yaml:50]: note also that any changes here need to also be applied to the preprod cp version as well

Comment (davidmiddletonmoj) [reply helm_deploy/values-dev.yaml:50]: Thanks @petergphillips will take you up on that offer this afternoon

File: helm_deploy/values-dev.yaml [hunk 4]
```diff
@@ -47,6 +47,13 @@ generic-service:
...
+      # Disable rule 942360 (SQL Injection) for url /api/bookings/:bookingId/alert/:alertId
+      SecRule REQUEST_URI "@rx ^\/api\/bookings\/[\w|\d]*\/alert\/[\w|\d|-]*" "id:800001,phase:1,pass,t:none,nolog,ctl:ruleRemoveById=942360"
+      # Disable rule 920420 (content-type = text/plain) for url /api/appointments/:appointmentId/comment
+      SecRule REQUEST_HEADERS:Content-Type "@rx ^text/plain" "id:800002,phase:1,pass,t:none,nolog,ctl:ruleRemoveById=920420,chain"
```
[the remark this answers is not in the document]
Comment (davidmiddletonmoj) [reply helm_deploy/values-dev.yaml:55]: Awesome, will remove that then

File: helm_deploy/values-dev.yaml (hunk 3 above)
Comment (davidmiddletonmoj) [reply helm_deploy/values-dev.yaml:51]: Yeah, this started from Dom Bull getting 406 errors updating alerts.

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**